source/uwp/AdaptiveCardTestApp/Expected/v1.0.windows-notification.v1.0.Elements.ColumnSet.Input.Text.VerticalStretch.json

```diff
@@ -0,0 +1 @@
+{"HostConfigHash":"f7dd633","CardHash":"1febc8e","Error":null}
```

Comment:
Looks like maybe some of these files are left over from the v1.0 version, please remove